### ISSUE TITLE
mvn test (without clean) gives 'io.github.blackrock:protoc-maven-plugin:2.0.0:run failed: Cannot read the array length because "<local6>" is null'

### ISSUE THREAD

**Issue by cmardini** [blackrock/protoc-jar-maven-plugin#25]
**Describe the bug**
In my project, when the protos aren't generated into sources but I try to run tests with 'mvn test', I get:

`Execution default of goal io.github.blackrock:protoc-maven-plugin:2.0.0:run failed: Cannot read the array length because "<local6>" is null`

But adding `clean` to the cmd line, I don't get this issue.

My plugin section looks like:
```
            <plugin>
                <groupId>io.github.blackrock</groupId>
                <artifactId>protoc-maven-plugin</artifactId>
                <version>2.0.0</version>
                <executions>
                    <execution>
                        <phase>generate-sources</phase>
                        <goals>
                            <goal>run</goal>
                        </goals>
                        <configuration>
                            <optimizeCodegen>true</optimizeCodegen>
                            <includeMavenTypes>direct</includeMavenTypes>
                            <includeStdTypes>true</includeStdTypes>
                            <includeDirectories>
                                <include>src/main/protobuf/include</include>
                                <include>src/main/protobuf/a-proto</include>
                            </includeDirectories>
                            <inputDirectories>
                                <include>src/main/protobuf/c-proto</include>
                                <include>src/main/protobuf/b-proto</include>
                                <include>src/main/protobuf/a-proto</include>
                            </inputDirectories>
                            <outputTargets>
                                <outputTarget>
                                    <type>java</type>
                                </outputTarget>
                                <outputTarget>
                                    <type>grpc-java</type>
                                    <pluginArtifact>io.grpc:protoc-gen-grpc-java:${grpc.version}</pluginArtifact>
                                </outputTarget>
                            </outputTargets>
                        </configuration>
                    </execution>
                </executions>
            </plugin>
```

**Environment (please complete the following information):**
 - CentOS 7
 - Maven 3.9.1

**Additional context**
Using 2.0.0

**Comment by spensonshih** [comment]
Thanks @cmardini , I will take a look at this.

**Comment by spensonshih** [comment]
This has been resolved in patch release 2.0.1.
